ddtrace/internal/telemetry/data/request_body.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+import json
+from typing import Optional
+
+from ...runtime import get_runtime_id
+from .application import Application
+from .host import Host
+from .payload import Payload
+
+
+class RequestBody:
+    def __init__(self, payload, application=None, host=None):
+        self.tracer_time = int(datetime.now().timestamp())  # type: int
+        self.runtime_id = get_runtime_id()  # type: str
+        self.api_version = "v1"  # type: str
+        self.seq_id = 0  # type: int
+        self.application = application or Application()  # type: Application
+        self.host = host or Host()  # type: Host
```

Comment:
Do `Application` and `Host` have to be a class here? They seem like static data and the classes aren't going to encode with `json.dumps()` properly.